k8s/tailscale/.github/workflows/staging.yaml

```diff
@@ -0,0 +1,43 @@
+name: Deploy to Massdriver
+on:
+  push:
+    branches: [main]
+
+jobs:
+  push_and_deploy:
+    runs-on: ubuntu-latest
+    env:
+      MASSDRIVER_ORG_ID: ${{ secrets.MASSDRIVER_ORG_ID }}
+      MASSDRIVER_API_KEY: ${{ secrets.MASSDRIVER_API_KEY }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Massdriver CLI
+        uses: massdriver-cloud/actions@v4
+      - name: Push Image
+        uses: massdriver-cloud/actions/image_push@v4
+        with:
+          namespace: massdrivercloud
+          image-name: express-mongoose-demo
+          artifact: ${{ secrets.MASSDRIVER_ARTIFACT_ID }}
```

Comment:
@coryodaniel could you add repo secrets for `MASSDRIVER_ARTIFACT_ID`, `MASSDRIVER_ORG_ID`, and `MASSDRIVER_API_KEY` for the sandbox org?